lib/hotelsystem.rb

```diff
@@ -0,0 +1,80 @@
+require_relative "reservation"
+
+module Hotel
+  class HotelSystem
+    attr_reader :rooms, :reservations
+
+    def initialize(reservations = [])
+      @rooms = [*1..20]
+      @reservations = reservations
+    end
+
+    def list_rooms
+      room_list = @rooms.map do |num|
+        "Room ##{num}"
+      end
```

Comment:
When we said "access the list of all of the rooms in the hotel", we meant being able to call a method and get back an array of whatever your program uses to represent rooms. The `attr_accessor :rooms` above accomplishes this, and this method is not needed.